cfme/tests/ssui/test_ssui_dashboard.py

```diff
@@ -6,51 +6,66 @@
 import pytest
 
 import cfme.intelligence.chargeback.assignments as cb
-import cfme.intelligence.chargeback.rates as rates
 from cfme import test_requirements
 from cfme.infrastructure.provider import InfraProvider
 from cfme.infrastructure.provider.scvmm import SCVMMProvider
 from cfme.markers.env_markers.provider import ONE_PER_TYPE
 from cfme.services.dashboard import Dashboard
 from cfme.utils.appliance import ViaSSUI
-from cfme.utils.blockers import GH
 from cfme.utils.log import logger
 from cfme.utils.version import current_version
 from cfme.utils.wait import wait_for
 
 
 pytestmark = [
-    pytest.mark.meta(server_roles="+automate", blockers=[GH('ManageIQ/integration_tests:7479')]),
+    pytest.mark.meta(server_roles="+automate"),
     pytest.mark.usefixtures('uses_infra_providers'),
     test_requirements.ssui,
     pytest.mark.long_running,
     pytest.mark.ignore_stream("upstream"),
-    pytest.mark.provider([InfraProvider], selector=ONE_PER_TYPE,
-                         required_fields=[['provisioning', 'template'],
-                                          ['provisioning', 'host'],
-                                          ['provisioning', 'datastore']],
-                         scope="module"),
+    pytest.mark.provider(
+        [InfraProvider],
+        selector=ONE_PER_TYPE,
+        required_fields=[['provisioning', 'template'],
+                         ['provisioning', 'host'],
+                         ['provisioning', 'datastore']],
+        scope="module"
+    ),
 ]
 
 
 @pytest.fixture(scope="module")
-def new_compute_rate(enable_candu):
+def enable_candu(appliance):
```

Comment:
Required:
can you please try global fixture
https://github.com/ManageIQ/integration_tests/blob/master/cfme/fixtures/candu.py#L8

Reply:
Good find, but they don't quite match. This one disables automate and smartstate roles, the global one does not.

Reply:
@mshriver:
In `C&U` testing we tried to avoid load on appliance by putting other roles `ON` like `samrtstate`. That what my understanding but may be I am wrong @nachandr can help us to understand more.

Reply:
I'll leave that for @nachandr to address in another PR, for now since the methods are different, I'm going to leave this fixture in the module.

Reply:
@nachandr passing to you; please come with PR if you like to remove this fixture. I am sure for `c&u` we don't want these roles.